addons/point_of_sale/static/lib/idb-keyval/idb-keyval.js

```diff
@@ -0,0 +1,81 @@
+//     idb-keyval.js 3.2.0
```

Comment:
@pimodoo This library is also used in `website_event_track` module. Do you think I can put this in `web` module so that it can be shared in other modules?

Reply:
yes, it could be great, but I don't know if framework is ok with that

Reply:
I'll ask them.

Reply:
thx